README.md

```diff
@@ -1,3 +1,38 @@
-# canonicalwebteam.docs
+# canonicalwebteam.discourse_docs
 
 Flask extension to integrate discourse content generated to docs to your website.
+
+## Install
+
+Install the project with pip: `pip install canonicalwebteam.discourse_docs`
+
+You can add the extension on your project:
+
+```
+from canonicalwebteam.discourse_docs import DiscourseDocs, DiscourseAPI
+
+discourse_api = DiscourseAPI(
+    base_url="https://forum.snapcraft.io/",
+    frontpage_id=3781,  # The "Snap Documentation" topic
+    category_id=15,  # The "doc" category
+)
+
+# From constructor
+DiscourseDocs(
+    app=app,
+    model=discourse_api,
+    url_prefix="/",
+    document_template="document.html",
```

Comment:
In general, both the default `url_prefix` and the default `document_template` should be used. I think the wording here encourages people to overwrite the default, which is probably not what we want is it?